guides/common/modules/con_renaming-server-or-smart-proxy.adoc

```diff
@@ -1,17 +1,9 @@
-[id="Renaming_Server_or_Smart_Proxy_{context}"]
-= Renaming {ProjectServer} or {SmartProxyServerTitle}
+[id="Renaming_{project-context}_Server_or_{smart-proxy-context}_{context}"]
+= Renaming {ProjectServerTitle} or {SmartProxyServerTitle}
 
-To rename {ProjectServer} or {SmartProxyServer}, you must use the `{project-change-hostname}` script.
+To rename {ProjectServer} or {SmartProxyServer}, use the `{project-change-hostname}` script.
 
-If you rename {ProjectServer}, you must reregister all {Project} clients and configure each {SmartProxyServer} to point them to the new {Project} host name.
-If you use custom SSL certificates, you must regenerate them with the new host name.
-If you use virt-who, you must update the virt-who configuration files with the new host name.
-
-If you rename {SmartProxyServer}, you must reregister all {SmartProxy} clients and update the {SmartProxy} host name in the {ProjectWebUI}.
-If you use custom SSL certificates, you must regenerate them with the new host name.
-
-[WARNING]
+[IMPORTANT]
 ====
-The renaming process shuts down all {ProjectServer} services on the host being renamed.
-When the renaming is complete, all services are restarted.
+When changing the domain name of your {ProjectServer} or {SmartProxyServer}, update the hostname using `{project-change-hostname}` as well to avoid networking issues.
```

Comment:
```suggestion
When changing the domain name of your {ProjectServer} or {SmartProxyServer}, update the hostname using `{project-change-hostname}` to avoid networking issues.
```